ansible/README.md

```diff
@@ -95,34 +93,63 @@ It's up to you how you create these VMs - kvm on a bare metal machine, VM on a c
 
 Ensure that your VMs have IP addresses that do not change.
 
-## Configuring virtual machines
+## Preparing to run ansible
 
 ### All VMs
 
 Copy the example hosts file:
 
 `cp hosts.example.ini hosts.ini`
 
-* replace the `ansible_host` values (`X.X.X.X`) with the IPs that you can reach by SSH.
-* replace the `ip` values (`Y.Y.Y.Y`) with the IPs which you wish kubernetes to provide services on.
+* Edit the hosts.ini, setting the permanent IPs of the hosts you are setting up wire on. 
+ * replace the `ansible_host` values (`X.X.X.X`) with the IPs that you can reach by SSH. these are the 'internal' addresses of the machines, not what a client will be connecting to.
+ * replace the `ip` values (`Y.Y.Y.Y`) with the IPs which you wish kubernetes to provide services to clients on.
+
+There are more settings in this file that we will set in later steps.
 
 #### WARNING: host re-use
 
-The playbooks mess with the hostnames of their targets.  You MUST pick different (virtual) hosts for the different playbooks.  If you e.g. want to run C* and k8s on the same 3 machines, the hostnames will be overwritten by the second installation playbook, corrupting the first.
+Some of these playbooks mess with the hostnames of their targets.  You MUST pick different (virtual) hosts playbooks that rename the host.  If you e.g. attempt to run Cassandra and k8s on the same 3 machines, the hostnames will be overwritten by the second installation playbook, corrupting the first.
 
-#### Authentication
+FIXME: what playbooks are guilty of this?
```

Comment:
https://github.com/wireapp/wire-server-deploy/blob/master/ansible/cassandra.yml#L10 - the [hostname role](https://github.com/ANXS/hostname)